packages/studioCMS/src/integrations/studioCMSDashboard/routes/dashboard/pages/configuration/admins.astro

```diff
@@ -103,9 +117,13 @@ const pageTitle = `Site Config | ${contextConfig.title}`;
                         {
                             adminList.map((admin: typeof Permissions.$inferSelect) => {
                                 return (
-                                    <tr class="bg-white border-b dark:bg-gray-800 dark:border-gray-700">
-                                        <th scope="row" class="px-6 py-4 font-medium text-gray-900 whitespace-nowrap dark:text-white">{admin.username}</th>
-                                        <td class="px-6 py-4">{admin.rank}</td>
+                                    <tr class="bg-base-200 text-inherit">
+                                        <th scope="row" class="px-6 py-4 font-medium whitespace-nowrap">{admin.username}</th>
+                                        {
+                                            admin.rank === 'admin' ? 
+                                            <td class="px-6 py-4 text-yellow font-bold drop-shadow-md">{toPascalCase(admin.rank)}</td> :
```

Comment:
I think it's really funny that `toPascalCase` isn't pascal cased 😂 

I don't think it should be btw, this is just a joke